ios/FluentUI/Core/Theme/Tokens/ShadowInfo.swift

```diff
@@ -59,4 +60,13 @@ public struct ShadowInfo {
 
     /// The vertical offset of the shadow for shadow 2.
     public let yTwo: CGFloat
+
+    /// Applies the chosen shadow to a specified view
+    public func applyShadow(for view: UIView, isShadowOne: Bool) {
```

Comment:
Why only apply one shadow? If our designs have two shadows, shouldn't we just apply both here, rather than needing to call this twice?

Reply:
I don't think we can apply both shadows like that. Essentially the layer will just use the latest set values. Also I believe for most of our shadows we don't use both. It's usually either the elevation shadow or the border one.

Reply:
If most of our shadows don't use both shadows, then `ShadowInfo` shouldn't include both?

Reply:
Talked offline and it seems both shadows need to be applied for the controls. I will work on it.

Reply:
+1 on applying both shadows. AFAIK, it's very intentional in design to _most of the time_ want both